tests/10-create.js

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const chai = require('chai');
+const {implementations} = require('vc-api-test-suite-implementations');
+const {validVC} = require('../credentials');
+const {
+  checkDataIntegrityProofFormat
+} = require('data-integrity-test-suite-assertion');
+const {
+  filterMap,
+  getPublicKeyBytes,
+  bs58Decode
+} = require('./helpers');
+
+const predicate = ({value}) =>
+  value.issuers.some(issuer => issuer.tags.has('Ed25519Signature2020'));
+// only use implementations that use `Ed25519 2020`
+const filtered = filterMap({map: implementations, predicate});
+const should = chai.should();
+const bs58 = /^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]+$/;
+
+// multiple test suite names violate max-len
+/* eslint-disable max-len */
+
+describe('Ed25519Signature2020 (create)', function() {
+  for(const [name, implementation] of filtered) {
+    let verifier;
+    let issuedVC;
+    let proofs;
+    before(async function() {
+      const issuer = implementation.issuers.find(issuer =>
+        issuer.tags.has('Ed25519Signature2020'));
+      verifier = implementation.verifiers.find(verifier =>
+        verifier.tags.has('VC-HTTP-API'));
```

Comment:
```suggestion
        verifier.tags.has('VC-API'));
```

This should probably be all lowercase... and might need to be ['vc-verifier-api'] or ['vc-api', 'verifier'] ... probably the former. I understand that there is a problem with stuff like 'Ed25519Signature2020' ... in the future, the cryptosuites are going to be all lowercase e.g., 'eddsa-2022'

Reply:
this one will require changes in other libraries and will cause issues with other tests suites as this is set in implementations: https://github.com/w3c-ccg/vc-api-test-suite-implementations/tree/initial/implementations

We might want to make an issue on implementations and then update all test suites that use that tag (I believe the verifier tests also use it).

p.s. we need consistent and agreeable tags for implementations.